game_of_life/src/GameOfLife/GameOfLife.js

```diff
@@ -91,16 +89,17 @@ export default class GameOfLife extends React.Component {
     computeAliveNeighboursAmount(columnIndex, rowIndex) {
         let aliveNeighboursAmount = 0;
 
-        const neighbourOffsets = [
-            [-1, 0], // left
-            [-1, 1], // top left
-            [0, 1], // top
-            [1, 1], // top right
-            [1, 0], // right
-            [1, -1], // bottom right
-            [0, -1], // bottom
-            [-1, -1], // bottom left
-        ];
+        const neighbourOffsets = [];
+
+        // Calculating the relative coordinates of neighbours
+        for(let i = -1; i <= 1; i++) { // horizontal offset
+            for(let j = -1; j <= 1; j++) { // vertical offset
```

Comment:
I would rather call those variables `horizontalOffset` or `columnOffset` instead of explaining that in a comment :)